tests/integration/responses/test_mcp_authentication.py

```diff
@@ -0,0 +1,153 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+import os
+
+import pytest
+
+from llama_stack.core.library_client import LlamaStackAsLibraryClient
+from tests.common.mcp import make_mcp_server
+
+from .helpers import setup_mcp_tools
+
+# Skip these tests in replay mode until recordings are generated
+pytestmark = pytest.mark.skipif(
+    os.environ.get("LLAMA_STACK_TEST_INFERENCE_MODE") == "replay",
+    reason="No recordings yet for authorization tests. Run with --inference-mode=record-if-missing to generate.",
+)
+
+
+def test_mcp_authorization_bearer(compat_client, text_model_id):
```

Comment:
can you replace `compat_client` by `responses_client` and remove the LibraryClient skip -- things changed around this due to https://github.com/llamastack/llama-stack/pull/4146